corehq/apps/hqwebapp/views.py

```diff
@@ -96,9 +104,14 @@ def celery_check():
         else:
             chk = (True, None)
     except IOError as e:
+        if is_deploy_in_progress():
+            return (True, 'Currently down, but deploy is in progress')
```

Comment:
not a huge deal, but elsewhere in this method `chk` is set instead of immediately returning.